workshop_2018_2019/angular_workshop.md

```diff
@@ -2,22 +2,22 @@
 Angular is front-end frame webframework mostly used to build single page applications. For this workshop, we will be using Angular to show your brain-art in the browser.
 
 ## Install Node.js, npm, Angular
-You must have Node.js and npm package manager installed in your machine before getting started with angular. After you have those two, install angular.
+You must have Node.js and npm package manager installed in your machine before getting started with angular. After you have those two, install angular. Node.js and npm can be installed from here: https://nodejs.org/en/download/. To install angular: run this command in your terminal (no quotes): "npm install -g angular-cli"
 
 Follow the instructions here to set up your environment: https://angular.io/guide/setup-local
 
-Step 2 and Step 3 in above tutorial is about creating your first angular project. Complete them and move on to basic Angular tutorial.
+To make your workshop experience as useful as possible, make sure you have angular setup and ready to go at the start of the workshop (this means installing everything you need beforehand!), because you will be diving straight into writing your first angular application!
 
 ## Basic Angular tutorial
-To be able to use Angular, it is essential that you have the very basic idea of HTML, CSS, JavaScript/TypeScript. If you are not familiar with it already, go to next step then come back to this one.
+To be able to use Angular, it is essential that you have the very basic idea of HTML, CSS, JavaScript/TypeScript. If you are not familiar with these already, we highly recommend that you go through the resources below for HTML, CSS, JavaScript before doing the angular tutorial.
```

Comment:
- I'd say start off with HTML, CSS, etc first instead of telling them to go to the next section
- also make this a journey for them; ie instead of "it is essential...", possibly use "Mastering HTML, CSS, JS is the first step towards getting epic at Angular!"